context/app/static/js/components/detailPage/provenance/ProvSection/ProvSection.jsx

```diff
@@ -1,29 +1,31 @@
 import React from 'react';
+
 import { useAppContext, useFlaskDataContext } from 'js/components/Contexts';
 import useProvData from 'js/hooks/useProvData';
 import { Alert } from 'js/shared-styles/alerts';
 import SectionHeader from 'js/shared-styles/sections/SectionHeader';
 import { DetailPageSection } from 'js/components/detailPage/style';
 import ProvTabs from '../ProvTabs';
 
-function ProvSection({ iconTooltipText }) {
+function ProvSection() {
   const {
     entity: { uuid, entity_type },
   } = useFlaskDataContext();
   const { groupsToken, entityEndpoint } = useAppContext();
   const { provData, isLoading } = useProvData(uuid, entityEndpoint, groupsToken);
+  const provenanceTooltipText = `The provenance shows the sequence of events and actions that led to this page creation.`;
```

Comment:
Anything variable that doesn't depend on props or variables defined inside of the component function can be pulled outside the component. This avoids the variables being recreated on each re-render.

Reply:
I believe this is what you meant:

![image](https://github.com/hubmapconsortium/portal-ui/assets/45410262/399c897a-59fe-4241-9b0a-6e278d910ebe)